testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/DefaultGradleDistribution.groovy

```diff
@@ -20,10 +20,52 @@ import org.gradle.api.internal.artifacts.ivyservice.CacheLayout
 import org.gradle.cache.internal.CacheVersion
 import org.gradle.test.fixtures.file.TestDirectoryProvider
 import org.gradle.test.fixtures.file.TestFile
+import org.gradle.tooling.internal.consumer.DefaultGradleConnector
 import org.gradle.util.GradleVersion
 
 class DefaultGradleDistribution implements GradleDistribution {
-    private static final String DISABLE_HIGHEST_JAVA_VERSION = "org.gradle.java.version.disableHighest";
```

Comment:
I removed this since it did not seem to be in use anywhere

Reply:
The only reference removed in this commit below
https://github.com/gradle/gradle/pull/33562/files#diff-d9e2403c30cf163e9b413888e99b845f9c8036c2aa61579060b138cfa9213248L147